fees/floki-trading-bot.ts

```diff
@@ -0,0 +1,47 @@
+import { FetchOptions, SimpleAdapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { queryDune } from "../helpers/dune";
+
+const duneQueryId: any = {
+  [CHAIN.ETHEREUM]: '4082746',
+  [CHAIN.BSC]: '4082749',
+  [CHAIN.BASE]: '4082753',
+}
+
+async function fetchFees(options: FetchOptions) {
+  const dailyFees = options.createBalances();
+  const rows = (await queryDune(duneQueryId[options.chain], {
```

Comment:
pls use query dune sql instead, or if youre just counting money that goes to an address you can use the function we use in other tg bot adapters